src/org/loklak/api/search/QuoraProfileScraper.java

```diff
@@ -87,9 +102,40 @@ public static SusiThought  scrapeQuora(String profile) {
 		String rssFeedLink = url + "/rss";
 		quoraProfile.put("rss_feed_link", rssFeedLink);
 
+		Elements userBasicInfo = userHTML.getElementsByAttributeValueContaining("class", "UserCredential IdentityCredential");
+		for (Element info: userBasicInfo) {
+			String infoText = info.text();
+			if (infoText.startsWith("Works")) {
+				quoraProfile.put("works_at", infoText);
```

Comment:
@djmgit You can grab the element `<div class="AboutSection">`, this element has exactly four `span` tags with `id` attribute. So those can be collected and then we need to check if they contain a `span` tag with `class` attribute that has value **UserCredential**, if yes extract the text and assign it to relevant JSON key. 
Something like this -> https://pastebin.ubuntu.com/24718646/ .